src/Contexts/Adaptors/EVMAdaptors/EVMDestinationAdaptorProvider.tsx

```diff
@@ -250,11 +250,13 @@ export const EVMDestinationAdaptorProvider = ({
 
   useEffect(() => {
     if (
+      process.env.REACT_APP_TRANSFER_FALLBACK_ENABLED === "true" &&
```

Comment:
Let's move this complex condition to a variable with a good name and use it inside the if

Reply:
Fixed